<launch>

<node pkg="robot_localization" type="ekf_localization_node"
      name="robot_localization_ekf_node_map"
      clear_params="true">
  <param name="frequency" value="10" />  
  <param name="sensor_timeout" value="1" />  
  <param name="two_d_mode" value="true" />
  <param name="publish_tf" value="true" />
  <param name="map_frame" value="map" />
  <param name="odom_frame" value="robot_odom_base" />
  <param name="base_link_frame" value="robot_link" />
  <param name="world_frame" value="robot_odom_base" />

  <param name="odom0" value="/wheel_encoders/odom" />
  <rosparam param="odom0_config">[true, true, true,
                                  false, false, true,
                                  true, true, false,
                                  false, false, true,
                                  false, false, false]</rosparam>
  <param name="odom0_differential" value="true"/>
<!--
  <param name="imu0" value="/imu/data" />
  <rosparam param="imu0_config">[false, false, false,
                                 false, false, false,
                                 false, false, false,
                                 true, true, true,
                                 true, true, true]</rosparam>
  <param name="imu0_differential" value="true"/>
  <param name="imu0_remove_gravitational_acceleration" value="false"/>
-->
 </node>

</launch>

